packages/flutter/test/cupertino/text_field_test.dart

```diff
@@ -2190,14 +2190,14 @@ void main() {
 
     expect(lastCharEndpoint.length, 1);
     // The last character is now on screen.
-    expect(lastCharEndpoint[0].point.dx, moreOrLessEquals(786.73, epsilon: 0.01));
+    expect(lastCharEndpoint[0].point.dx, moreOrLessEquals(786.73, epsilon: 2));
```

Comment:
Not sure why but I had to change the epsilon here so these tests pass.

Reply:
Weird, nothing jumps out at me as a cause.  It is a big epsilon, but if no golden tests are failing then maybe it's not a concern...

Reply:
I reverted the epsilon changes. Tests are passing.